coverage.py

```diff
@@ -393,7 +393,7 @@ def process_batch_and_handle_results(self, batch):
         num_ignored = 0
         records = []
 
-        unhandled_items = set(batch)
+        unhandled_items = batch
```

Comment:
I'm a little concerned about this change because I think it is possible for a batch to contain the same item twice. If that happens, whatever service is processing the items will probably return only a single status for that item. We'll call `unhandled_items.remove(item.obj)` once and `item` will still be in `unhandled_items` once we're done, making it look like the service just ignored that item.

If it's not possible to turn this into a set in Python 3, can we build a de-duped list instead?

Reply:
I'll check this again.

Reply:
I'm not sure why it failed before but I added it back now.